lib/sequenceserver/blast/error.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'constants'
+require_relative 'exceptions'
+
+module SequenceServer
+  module BLAST
+    class Error
+      # Catches errors encountered during the search
```

Comment:
Comments usually precede the class, module, method, expression they are meant for.